### PR TITLE
7865: Return activity feed page header to a san-serif font

### PR DESCRIPTION
**Pull request by frankhereford**
## Associated issues

This PR intends to address the paticular H2 seen in https://github.com/cityofaustin/atd-data-tech/issues/7865 but not to close the issue.

## Testing
**URL to test:** 
https://deploy-preview-488--atd-moped-main.netlify.app/moped/

**Steps to test:**

View a project's activity feed and observe the header.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

**Comment by amenity** [comment]
Thanks for looking at this, @frankhereford! Can we use the MUI classes to ensure it gets the same styles as all the other h2s?

Screen Shot 2021-12-02 at 12 02 17 PM

**Comment by frankhereford** [comment]
> Thanks for looking at this, @frankhereford! Can we use the MUI classes to ensure it gets the same styles as all the other h2s?

For sure! I'll make those changes later today after work when I get home. Thanks for the feedback! I'll re-request your review when it's in there.

**Comment by frankhereford** [comment]
@amenity identified that the real issue at hand here is not with this particular H2 but potentially something more complex and it'll be a better long term thing for the app to work it in that context. Thanks!

**Comment by frankhereford** [comment]
@amenity and I had further discussion, and she clarified that this change is still something she'd like to see done but to not let it close the issue itself, as it can be our ticket for a potential font issue that may be in effect.

**Review by amenity — APPROVED** [review]

**Comment by amenity** [comment]
Looks good!

**Comment by frankhereford** [comment]
@amenity - thank you for your review. I've added the rest of the dev team to get a code review and look forward to merging this. Thanks!

File: moped-editor/src/views/projects/projectView/ProjectActivityLog.js [hunk 1]
```diff
@@ -187,7 +191,7 @@ const ProjectActivityLog = () => {
   return (
     <ApolloErrorHandler error={error || lookupError}>
       <CardContent>
-        <h2 style={{ padding: "0rem 0 2rem 0" }}>Activity feed</h2>
+        <h2 className={classes.projectPageHeader} style={{ padding: "0rem 0 2rem 0" }}>Activity feed</h2>
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectActivityLog.js:194]: do you think we could fold the padding style into the `projectPageHeader` style?

Comment (frankhereford) [reply moped-editor/src/views/projects/projectView/ProjectActivityLog.js:194]: I think we absolutely could and should - thank you for suggesting that. I'll amend the PR here momentarily.

**Review by sergiogcx — APPROVED** [review]

**Comment by mateoclarke** [comment]
👍

**Comment by chiaberry** [comment]
Maybe this was obvious to everyone else, but I figured out why this particular h2 was serifed. Our other h2s are `<Typography variant="h1" component="h2">` and not `<h2>`.

**Comment by frankhereford** [comment]
> Maybe this was obvious to everyone else, but I figured out why this particular h2 was serifed. Our other h2s are `<Typography variant="h1" component="h2">` and not `<h2>`.

@chia - thank you! I did not realize this, thank you for pointing it out. @amenity - would you like me to make a really quick pass back on this to align it with others?

**Pull request merged**